src/System.Diagnostics.Process/src/System/Diagnostics/Process.OSX.cs

```diff
@@ -2,8 +2,15 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using Microsoft.Win32.SafeHandles;
 using System.Collections.Generic;
 using System.ComponentModel;
+using System.Globalization;
```

Comment:
I do'nt think you need all those. I believe VS has a refactoring to remove unnecessary using statements.